### PR TITLE
added inorder tree traversal

### PR DESCRIPTION
**Pull request by shubhangi-singh21**
Inorder traversal added. Will be adding more traversals too. Pls check @siddharth25pandey .

### CODE REVIEW

**Comment by siddharth25pandey** [comment]
@shubhangi-singh21 Please edit the README file too with your name in Contributor's List

**Comment by shubhangi-singh21** [comment]
> @shubhangi-singh21 Please edit the README file too with your name in Contributor's List

Sure, i'll do that after adding 2 more traversal techniques. Will that be okay?

**Comment by siddharth25pandey** [comment]
> > @shubhangi-singh21 Please edit the README file too with your name in Contributor's List
> 
> Sure, i'll do that after adding 2 more traversal techniques. Will that be okay?

@shubhangi-singh21 Yeah fine, no worries

File: Trees Algorithm/binary_tree_traversal_inorder.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+using namespace std;
+
+void Inorder( struct node *root)
+{
+  if(root==0)
+  { return 0;}
+ Inorder( root->left);
+ printf("%d", root->data);
+ Inorder( root->right);
+}
```
Comment (siddharth25pandey) [inline Trees Algorithm/binary_tree_traversal_inorder.cpp:14]: Don't only add the part of the code.
We need a full working code with proper user friendly interface

File: Trees Algorithm/binary_tree_traversal_preorder.cpp [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+using namespace std;
+
+void Preorder( struct node *root)
+{
+  if(root==0)
+  { return 0;}
+ printf("%d", root->data);
+ Preorder( root->left);
+ Preorder( root->right);
+}
```
Comment (siddharth25pandey) [inline Trees Algorithm/binary_tree_traversal_preorder.cpp:14]: Don't only add the part of the code.
We need a full working code with proper user friendly interface

File: Trees Algorithm/binary_tree_traversal_inorder.cpp [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+void main()
+{
+  struct node *root;
+  printf( "Inorder is:");
+  Inorder(root);
```
Comment (siddharth25pandey) [inline Trees Algorithm/binary_tree_traversal_inorder.cpp:20]: Take the Input from the user

File: Trees Algorithm/binary_tree_traversal_preorder.cpp [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+void main()
+{
+  struct node *root;
+  printf( "Preorder is:");
+  Preorder(root);
+}
```
Comment (siddharth25pandey) [inline Trees Algorithm/binary_tree_traversal_preorder.cpp:21]: Take the Input from the user

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@shubhangi-singh21 Don't add only the part of the code.
We need a full working file.
Do the Suggested changes as required

**Comment by shubhangi-singh21** [comment]
@siddharth25pandey Made all changes. Pls check now.

**Review by siddharth25pandey — APPROVED** [review]
@shubhangi-singh21 LGTM
Able to merge
Thanks!

**Comment by siddharth25pandey** [comment]
@all-contributors please add @shubhangi-singh21  for her code

**Pull request merged**